nest/03.advanced_concepts.md

```diff
@@ -0,0 +1,78 @@
+# 3 Advanced concepts
+In this chapter, we will detail some advanced concepts related to `nest`, which will help understanding how `nest` works, but aren't strictly needed for a daily use of `nest`.
+Readers interested in `finest` should read this chapter, because those concepts are important to understand how `finest` works.
+
+## 3.1 Managing repositories
+### 3.1.1 The official repositories
+The official repositories are the ones maintained by the Raven-OS team.
+For now, only one is available: `stable`, which is the one enabled by default when installing Nest.
+It contains all the stable and safe-to-use packages for Nest, and is maintained by the Raven-OS team.
+
+### 3.1.2 The unofficial repositories
+There are also unofficial repositories for Nest which are not vetted by Raven-OS, so caution is advised.
+It is up to you to decide whether to trust their maintainers.
+
+### 3.1.3 Configuring repositories
+The repositories configuration is specified in the configuration file of Nest (`/etc/nest/config.toml`).
```

Comment:
```suggestion
The repositories' configuration is specified in the configuration file of Nest (`/etc/nest/config.toml`).
```
I'm not quite certain I got the right meaning, but I think this is the intended one.
```suggestion
The configuration of the repositories is found in the configuration file of Nest (`/etc/nest/config.toml`).
```